### ISSUE TITLE
Only one davice can access the bridge

### ISSUE THREAD

**Issue by thexperiments** [michbeck100/pimatic-hap#44]
I just deleted the hap-database and freshly set up pimatic-hap with my iPhone. Worked fine and it works with the iPhone on the local network like a charm.

However... It is not working on the iPad anymore. Looks like all devices synced to the iPad from iCloud without problems but in the debug output of the plugin I see that there eems to be something wrong with authentication.

```
22:05:12.863 [pimatic-hap] debug: [d9:33:c1:bc:8c:3f] HAP Request: POST /pair-verify
22:05:12.869 [pimatic-hap] debug: [d9:33:c1:bc:8c:3f] Pair verify step 1/2
```

It never gets to 2/2 and I can se the same output when starting my appleTV.

Both worked before I set it up fresh.

Any idea what that could be?

**Comment by michbeck100** [comment]
did the same today and both devices, iPhone and iPad worked. Can't tell what the problem might be, maybe the iCloud sync wasn't complete and the HomeKit data on your iPad was still outdated?

**Comment by thexperiments** [comment]
I think it is a problem of apple :( I just completely deleted the home from the phone.
I didn't do this before, before I just removed the homekit bridge from the home.

I now set everything up again and it works again with all iDevices locally and remote...

I really hope they get some of these homekit bugs fixed as Siri is really convenient in some situations and the Home app and iOs integration is quite nice.
